scenarios/ebb_and_flow.go

```diff
@@ -233,36 +228,80 @@ func (e *ebbAndFlow) run(ctx context.Context) error {
 				iter++
 			}
 		case <-backlogTicker.C:
-			e.Logger.Infof("Backlog: %d, target: %d, rate: %d/s, gen: %d, proc: %d",
-				backlog, target, rate, generated, processed)
+			e.Logger.Debugf("Backlog: %d, target: %d, rate: %d/s, started: %d, completed: %d",
+				backlog, target, rate, started, completed)
 		}
 	}
 
 	e.Logger.Info("Scenario complete; waiting for all workflows to finish...")
 	startWG.Wait()
 	e.Logger.Info("Verifying scenario completion...")
 
-	// Post-scenario: verify that at least one iteration was completed.
-	completedWorkflows := int(e.processedCount.Load())
-	if completedWorkflows == 0 {
-		return fmt.Errorf("no workflows completed")
+	e.stateLock.Lock()
+	totalCompletedWorkflows := int(e.state.TotalCompletedWorkflows)
+	e.stateLock.Unlock()
+
+	// Post-scenario: verify that at least one workflow was completed.
+	if totalCompletedWorkflows == 0 {
```

Comment:
This is now using the resume-aware counter; so it's not going to be zero if we crash and resume after the iterations were all done.